<ui version="4.0" >
 <class>EditCoverSearchDialog</class>
 <widget class="QDialog" name="EditCoverSearchDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>149</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777215</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="localeLabel" >
         <property name="text" >
          <string>Amazon Locale: </string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="KComboBox" name="amazonLocale" >
         <item>
          <property name="text" >
           <string>International</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Canada</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>France</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Germany</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Japan</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>United Kingdom</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="textLabel" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string comment="KDE::DoNotExtract" >TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KLineEdit" name="SearchLineEdit" />
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="KPushButton" name="SearchButton" >
         <property name="text" >
          <string>&amp;Search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="Bar" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons" >
          <set>QDialogButtonBox::Cancel</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>Bar</sender>
   <signal>accepted()</signal>
   <receiver>EditCoverSearchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>226</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Bar</sender>
   <signal>rejected()</signal>
   <receiver>EditCoverSearchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>311</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SearchButton</sender>
   <signal>clicked()</signal>
   <receiver>EditCoverSearchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>155</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel" >
     <x>125</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
